src/Illuminate/Database/Query/Grammars/Grammar.php

```diff
@@ -612,6 +612,20 @@ protected function compileUnion(array $union)
     }
 
     /**
+     * Compile an exists statement into SQL.
+     *
+     * @param \Illuminate\Database\Query\Builder $query
+     *
+     * @return string
+     */
+    public function compileExists(Builder $query)
+    {
+        $select = $this->compileSelect($query);
```

Comment:
It doesn't make a difference, the SQL server will ignore the `select` contents anyway when passed into `exists()`.